interfaces/input_parameters/ring_section.py

```diff
@@ -1,77 +1,260 @@
+# coding: utf8
+# Copyright 2014-2020 CERN. This software is distributed under the
+# terms of the GNU General Public Licence version 3 (GPL Version 3),
+# copied verbatim in the file LICENCE.md.
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+# Project website: http://blond.web.cern.ch/
+
+'''
+**Module gathering all general input parameters used for the simulation.**
+    :Authors: **Simon Albright**, **Alexandre Lasheen**
+'''
+
 # General imports
 import numpy as np
+import warnings
 
 # BLonD_Common imports
 from ...datatypes import datatypes as dTypes
-from ...datatypes import blond_function as bf
+from ...datatypes.blond_function import machine_program
```

Comment:
I'd prefer not to do it this way, but to stick with `import blond_function as XXX`

Reply:
The call is made so that machine_program can be imported from ring_section by the user